react-client/src/Components/DomSwitch.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const DomSwitch = () => {
+  return (
+    <div id = "divDeveloperTools"
+    style={{
```

Comment:
Please move the stylings to our `style.scss` file. Thanks.

Reply:
No. Dire na please. Thanks.

Reply:
@hnbmunoz Also this one para makatuon naka mag code sa SASS instead sa React for stylings.